doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md

```diff
@@ -115,21 +115,7 @@ The 3rd party conponents listed above have to be cross-compiled before building
 
     Copy `${SDKTARGETSYSROOT}/usr/local/lib/libboost_*` object files to Edison's `/usr/local/lib` directory (e.g. using SSH, SCP, etc.).
```

Comment:
`SDKTARGETSYSROOT` is defined in `environment-setup-core2-32-poky-linux` file which is generated by installer.